src/pages/home/sidebar/BottomTabAvatar.tsx

```diff
@@ -68,20 +67,18 @@ function BottomTabAvatar({isCreateMenuOpen = false, isSelected = false}: BottomT
     }
 
     return (
-        <Tooltip text={translate('initialSettingsPage.accountSettings')}>
-            <PressableWithFeedback
-                onPress={showSettingsPage}
-                role={CONST.ROLE.BUTTON}
-                accessibilityLabel={translate('sidebarScreen.buttonMySettings')}
-                wrapperStyle={styles.flex1}
-                style={[styles.bottomTabBarItem]}
-            >
-                {children}
-                <Text style={[styles.textSmall, styles.textAlignCenter, isSelected ? styles.textBold : styles.textSupporting, styles.mt0Half, styles.bottomTabBarLabel]}>
-                    {translate('common.settings')}
-                </Text>
-            </PressableWithFeedback>
-        </Tooltip>
+        <PressableWithFeedback
```

Comment:
@ZhenjaHorbach Since we have the bottom text inside `BottomTabAvatar`, I think removing the tooltip is fine. let me know what you think.
https://github.com/Expensify/App/blob/987ff1b536f2144e5fd492c6d2d2aa621d68442f/src/pages/home/sidebar/BottomTabAvatar.tsx#L81